moveit_ros/hybrid_planning/test/config/demo_controller.yaml

```
@@ -0,0 +1,28 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 500  # Hz
+    panda_arm_controller:
+      type: joint_trajectory_controller/JointTrajectoryController
+    joint_state_controller:
+      type: joint_state_controller/JointStateController
+    panda_joint_group_position_controller:
+      type: position_controllers/JointGroupPositionController
+
+panda_arm_controller:
+  ros__parameters:
```

Need the joint names to use the other controller

```suggestion
  ros__parameters:
      joints:
      - panda_joint1
      - panda_joint2
      - panda_joint3
      - panda_joint4
      - panda_joint5
      - panda_joint6
      - panda_joint7
```